packages/client/package.json

```diff
@@ -15,6 +15,7 @@
   ],
   "devDependencies": {},
   "dependencies": {
-    "axios": "^0.21.1"
+    "axios": "^0.21.1",
+    "@botpress/messaging-base": "0.0.1"
```

Comment:
Since this package only contains typings I think it should be in the devDependencies section!

Reply:
The package contains a emitter class that implements functions so it's not just a types package